packages/immutable-rootfs/30cos-immutable-rootfs/cos-setup-rootfs.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=cOS system early rootfs setup
+DefaultDependencies=no
+After=initrd-root-fs.target dev-disk-by\x2dlabel-COS_OEM.device
```

Comment:
I'd like to avoid having this dependency to `initrd-root-fs.target`. This is required because after the installation the only place where we have cloud-init files is in `/system/oem` which is part of the rootfs. I consider this a limitation, because it prevents us to run some cloud-configs before the rootfs is mounted. I think it would be nice to copy the `rootfs` stage of cloud init files to `COS_OEM` volume at installation time, this way we could have the `rootfs` stage that runs before mounting the actual root. I do not have a clear use case now for that, but I think this could be relevant for the semipersistent /etc  for example, it could be the place to hook before mounting the disk (COS_OEM could be some USB stick or remote location).